am-lib/src/test/java/uk/gov/hmcts/reform/amlib/helpers/TestConstants.java

```diff
@@ -16,15 +18,15 @@
 
 public final class TestConstants {
 
-    public static final String ACCESS_TYPE = "user";
+    public static final AccessorType ACCESSOR_TYPE = AccessorType.USER;
+    public static final AccessType ACCESS_MANAGEMENT_TYPE = AccessType.EXPLICIT;
```

Comment:
Rename to be ACCESS_TYPE. I hope @swalker125 that it will be removed in your second PR.